src/components/quiz/QuizList.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { QuizQuestion } from "../../../dto/quiz";
+import EditButton from "../common/EditButton.vue";
+import DeleteButton from "../common/DeleteButton.vue";
+
+const props = defineProps({
+  question: {
+    type: Object as () => QuizQuestion,
+    default: () => {},
+  },
+  questionArray: {
+    type: Array as () => QuizQuestion[],
+    default: () => [],
+  },
+});
+
+const emit = defineEmits(["addPoint", "deleteQuestion"]);
+
+function postQuiz() {
+  console.log(props.questionArray);
+}
+
+function deleteQuestion(index: number) {
+  emit("deleteQuestion", index);
+}
+
+function addPoint(point: number, id: string) {
+  emit("addPoint", point, id);
+}
+
+function pointsArray(point: number) {
+  let start: number = 0;
```

Comment:
can be bit shorter like `return Array.apply(null, Array(point)).map((_, i) => i)`